orion/benchmark.py

```diff
@@ -44,14 +44,18 @@
 PIPELINE_DIR = os.path.join(os.path.dirname(__file__), 'pipelines', 'verified')
 
 VERIFIED_PIPELINES = [
-    'arima', 'lstm_dynamic_threshold', 'azure', 'tadgan'
+    'arima', 'lstm_dynamic_threshold', 'azure', 'tadgan',
+    'aer', 'lstm_autoencoder', 'dense_autoencoder'
 ]
 
 VERIFIED_PIPELINES_GPU = {
     'arima': 'arima',
     'lstm_dynamic_threshold': 'lstm_dynamic_threshold_gpu',
     'azure': 'azure',
-    'tadgan': 'tadgan_gpu'
+    'tadgan': 'tadgan_gpu',
+    'aer': 'aer',
```

Comment:
Can you confirm gpu version is the same with CPU version? The same question for `dense_autoencoder`

Reply:
The GPU version is the same for `dense_autoencoder` since it only uses the `dense` layer.

@sarahmish mentioned that I can ignore the `gpu` version of the `aer ` model. The pipeline file will be the same once we upgrade to TF2.0.